includes/Core/Authentication/Google_Proxy.php

```diff
@@ -330,6 +354,33 @@ public function get_site_fields() {
 		);
 	}
 
+	/**
+	 * Gets metadata fields.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array Metadata fields array.
+	 */
+	public function get_metadata_fields() {
+		$metadata = array(
+			'supports'         => implode( ',', $this->get_supports() ),
+			'nonce'            => wp_create_nonce( self::ACTION_SETUP ),
+			'mode'             => '',
+			'hl'               => $this->context->get_locale( 'user' ),
+			'application_name' => self::get_application_name(),
+			'service_version'  => Feature_Flags::enabled( 'serviceSetupV2' ) ? 'v2' : '',
+		);
+
+		/**
+		 * Filters the setup mode.
+		 *
+		 * @since n.e.x.t
```

Comment:
This is missing a `$param` doc for the filtered variable. In WordPress hooks you can just put any variable name there, e.g. `$mode`, it doesn't have to match the name of what's really passed to it.

Reply:
Updated.